### PR TITLE
Allow admins to view users' emails

### LINKED ISSUE #1247: Show e-mail on the profile page if you're an admin
When logged in as an admin I you should be able to see the user's e-mail on their profile page. This lets the admin contact the user privately if necessary.

### PR DESCRIPTION
**Pull request by loganmeetsworld**
Fixes #1247

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

File: src/supermarket/app/views/api/v1/users/show.json.jbuilder [hunk 1]
```diff
@@ -3,6 +3,7 @@ json.name @user.name
 json.company @user.company
 json.github Array(@github_usernames)
 json.twitter @user.twitter_username
+json.email @user.email
```
Comment (robbkidd) [inline src/supermarket/app/views/api/v1/users/show.json.jbuilder]: Does this limit viewing a user's email in the JSON response to only admins? I couldn't find a permissions guard for this one.

**Comment by robbkidd** [comment]
@tas50 As the originator of the "show admins user emails" feature, what do you think about its inclusion in the JSON response for user info? Good feature parity to add if user role checking is easy? Not necessary if role checking there is difficult?

**Comment by robbkidd** [comment]
👍

**Pull request merged**